gcsweb/README.md

```diff
@@ -0,0 +1,26 @@
+`gcsweb` is a tiny web frontend to [GCS](https://cloud.google.com/storage/docs/) browsing that DOES NOT REQUIRE A GOOGLE LOGIN.
+
+See it in action: http://gcsweb.k8s.io/gcs/kubernetes-release/release/
+
+#### Problem
+
+`kubernetes` releases can be downloaded using direct API links to specific
+files. However, to browse all available files at
+https://console.cloud.google.com/storage/browser/kubernetes-release/release/
+or with `gsutil` people
+[need Google login](https://cloud.google.com/storage/docs/access-public-data).
+
+#### Solution
+
+Run a public web app that uses authenticated requests on backend to extract
```

Comment:
This does not use authenticated requests - it uses the public API to access public buckets.